### ISSUE TITLE
wrong description of rotate_left/right

### ISSUE THREAD

**Issue by MathieuDerelle** [boazsegev/combine_pdf#158]
the source code says
```
    # rotate the page 90 degrees counter clockwise
    def rotate_left

    # rotate the page 90 degrees clockwise
    def rotate_right
```

but from what I've tested, it's the opposite, rotate_left if rotating clockwize et rotate_right counter clockwize

**Comment by SlavaEremenko** [comment]
Confirming this issue!
Version 1.0.16

**Comment by boazsegev** [comment]
@MathieuDerelle , thank you for opening this issue.

@SlavaEremenko , thank you for confirming this issue.

Although I haven't tested this issue yet, I suspect that the biggest question I will have with this issue is "how to fix it?":

1. I can fix the documentation, making things a bit awkward, since `rotate_left` implies counterclockwise motion.

2. I can fix the API implementation, breaking current implementations that rely on the rotation direction.

I think this change / fix will require a version bump due to it being a breaking change.

I'll think about this some more and would be happy to read your input.

Kindly,
   Bo.

**Comment by MathieuDerelle** [comment]
I think `rotate_left` and `rotate_right` do not make much sense. 
If you imagine a circle, rotating left could mean
- rotate the point at the bottom of the circle to the left => clockwise
- rotate the point at the top of the circle to the left => counter clockwise

You could define those 2 methods : `rotate_clockwise` / `rotate_counter_clockwise`
and deprecate `rotate_left` / `rotate_right`

**Comment by SlavaEremenko** [comment]
I agree with both of you. Keep the old methods, deprecate them, fix the docs and create new ones: `rotate_cw` and `rotate_ccw`.
